data/processed/processed.txt

```diff
@@ -0,0 +1 @@
+processed - Directory for augmented files from the model
```

Comment:
I will suggest put markdown files instead of txt files for the data folders. And maybe put some samples in each folder.

Reply:
I will work on that and put samples in designated folders.